### DISCUSSION TITLE
(Already exists) Add the description in each node (or part of the description)

### DISCUSSION THREAD

**Discussion by mr-E** [Ylianst/MeshCentral#4453] [Ideas]
Will there be a way to append the description in the nodes (maybe at least 20 or 30 characters) using a small font?
I understand that space is quite scarce, but: 

Our host names are somewhat "cryptic":
* WYYMM[A-Z] indicating that it is a windows machine purchased in the year, month and consecutive letter.
* Depto##, or the area of the equipment and a sequential letter

We have added descriptions in each MeshCentral node which allows us to locate them from the filter; but it would be very useful to see part of that description in each node in the "My Devices" list.
Then we could quickly do a search within the browser and locate it quickly.

![node with description](https://user-images.githubusercontent.com/1486378/186780495-05ca1dd3-4bf1-4b02-aa4b-feb7d8a3220d.png)

Thank you for taking it into consideration!

**Comment by si458** [comment]
If you click the list view icon (top right) you will then get a cog you can click and select description which might be helpful

But I believe in the icon view, it currently doesn't display the descriptions

**Reply by mr-E** [reply]
Hey! that's pretty cool. That will help me a lot.
Thank you!

**Comment by PathfinderNetworks** [comment]
You can also go in to a device and click on the Device name.  A box will pop up where can change the name of the device to whatever you want.  I name our devices very similarly to how you are wanting.  Such as in this screen grab.  The actual device names are the CCPAASC... and the  - personname is what I add to identify who that device is assigned to.  
![mesh](https://user-images.githubusercontent.com/48607498/186851865-04627591-b2cc-4856-88fd-7291c364101a.jpg)

**Reply by mr-E** [reply]
That's great...
But, the strange thing is that (being an administrator) I can only edit the "server-side device name" in some groups, but not in all:

![no edit for some devices in groups](https://user-images.githubusercontent.com/1486378/186931224-f1698fcc-8631-47a7-8e02-8542ba88640b.png)

I think the cause is this option, It was enabled in my groups. I will disable it; I hope without consequences. :crossed_fingers: 

![image](https://user-images.githubusercontent.com/1486378/186932681-d2d7f01c-b160-4550-8eaf-e5515f5d2237.png)
---
Edit:
Yes, that activates the trick to edit the device name. Thank you!
